pkg/plugins/policies/core/xds/listeners.go

```diff
@@ -26,6 +26,7 @@ func GatherListeners(rs *xds.ResourceSet) Listeners {
 	listeners := Listeners{
 		Inbound:      map[core_rules.InboundListener]*envoy_listener.Listener{},
 		Outbound:     map[mesh_proto.OutboundInterface]*envoy_listener.Listener{},
+		Egress:       map[core_rules.InboundListener]*envoy_listener.Listener{},
```

Comment:
Egress always has only 1 listener, so this can be just `Egress *envoy_listener.Listener`